### PR TITLE
Cortex analyst tasty bytes

### PR DESCRIPTION
**Pull request by sfc-gh-jgriffith**
Cortex Analyst sample

### CODE REVIEW

**Comment by sfc-gh-cbaechtold** [comment]
@sfc-gh-jgriffith I added some comments above- source code all looks good, given that this is external facing I'd like just a little bit more instruction to be included in the README.md- it's just a sample so I don't want to go through the full step-by-step quickstart stuff, but just some basics around "stand up your SF notebook by following XYZ instructions in the docs" or "configure the connections.toml like XYZ" and then this will be good to merge

**Comment by sfc-gh-jgriffith** [comment]
@sfc-gh-cbaechtold Added some detail to make it clear what was automatically being done by the setup script and what steps and decisions the user needs to make.

File: samples/cortex-analyst/Cortex Analyst Tasty Bytes/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Cortex Analyst Demo Using TastyBytes Demo Data
```
Comment (sfc-gh-jgriffith) [inline samples/cortex-analyst/Cortex Analyst Tasty Bytes/README.md:1]: .

File: samples/cortex-analyst/Cortex Analyst Tasty Bytes/Cortex Analyst Notebook/environment.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name: app_environment
```
Comment (sfc-gh-jgriffith) [inline samples/cortex-analyst/Cortex Analyst Tasty Bytes/Cortex Analyst Notebook/environment.yml:1]: .

**Review by sfc-gh-cbaechtold — APPROVED** [review]
Looks good, thanks!

**Pull request merged**